src/nft/components/collection/Card.tsx

```diff
@@ -490,7 +490,7 @@ const InfoContainer = ({ children }: { children: ReactNode }) => {
 
 const TruncatedTextRow = styled(Row)`
   padding: 2px;
-  white-space: nowrap;
+  white-space: pre;
```

Comment:
- preserves the no-wrap behavior, and also preserves whitespaces (which we need to do if we want a blank row)
- This style is also only used for sell card details, so this doesn't affect other cards in the app.